MovieGallery/Pages/Index.cshtml

```diff
@@ -5,12 +5,10 @@
 
 }
 <head>
-</head>
-<body>
-
     <div style="background-color: white">
         <h1 class="text-center" style="background-color: darkred; color: yellow; "> Movie Gallery</h1>
     </div>
-
+</head>
```

Comment:
Added the code in head as the head tag was left empty.

Reply:
div is visible content.
head should only have not visible content.
I would not implement this change.
head should have title, css, javascript, etc, but not visible content.